opennms-webapp-rest/src/test/java/org/opennms/web/rest/v2/MonitoringLocationRestServiceIT.java

```diff
@@ -164,4 +172,68 @@ public void testEventOnCreationAndDeletion() throws Exception {
         // delete the one with polling packages
         sendRequest(DELETE, "/monitoringLocations/location2", 204);
     }
+
+    @Test
+    @Transactional
+    public void testLocationLimits() throws Exception {
+        // There is one default location
+        for (int i = 0; i < LOCATION_COUNT - 1; i++) {
+            Location loc = new Location();
+            loc.locationName = "Location-" + String.format("%05d", i);
+            loc.monitoringArea = "LocationArea-" + String.format("%05d", i);
```

Comment:
You're using `fomat` already:

```suggestion
            loc.locationName = String.format("Location-%05d", i);
            loc.monitoringArea = String.format("LocationArea-%05d", i);
```

Reply:
I have fixed this, it now looks like this:
           loc.setLocationName(String.format("Location-%05d",i));
            loc.setMonitoringArea(String.format("LocationArea-%05d", i))